packages/manager/modules/pci/src/components/project/flavors-list/flavors-list.service.js

```diff
@@ -187,10 +187,10 @@ export default class FlavorsList {
         isNew,
         flavors: [
           ...filteredAndRearrangedFlavors.filter((flavor) =>
-            flavor?.tagsBlob.includes(TAGS_BLOB.IS_NEW),
+            flavor?.tagsBlob?.includes(TAGS_BLOB.IS_NEW),
```

Comment:
Yes but it's going to be the same code since 
the second line is différent from the first one
(flavor) => !flavor?.tagsBlob?.includes(TAGS_BLOB.IS_NEW)